### ISSUE TITLE
Error with imporing SDK as Module in Android Studio

### ISSUE THREAD

**Issue by maniart** [bambuser/react-native-bambuser-broadcaster#27]
Hello world,

I'm working on integrating the SDK with an existing React Native app. My Android development environment is setup and I've been closely following steps [suggested in the docs](https://bambuser.com/docs/broadcasting/android/#add-the-broadcast-sdk).

With the top level `./android` directory of my React Native open in Android Studio, when I open the module settings for the top level directory, and under Module I choose Import Module and navigate to the `.aar` SDK file I downloaded, I get the following error in a dialog box.

```
Cannot Import
Cannot import anything from 
/path/to/libbambuser-0.9.20.aar
```

I have looked around and I can't find any obvious mistakes, or any posts about this issue elsewhere. Any ideas? 

Thanks!

**Comment by andreasbergqvist** [comment]
Not sure if it helps.
But for me it worked (did it today), just opened the React Native android folder in Android Studio (3.6) right clicked the app-folder and selected "Open module settings". Clicked the + and selected the .aar file.

**Comment by maniart** [comment]
Thanks @andreasbergqvist this is exactly what I did, and this is the response I get:

<img width="889" alt="Screen Shot 2020-03-27 at 8 23 48 AM" src="https://user-images.githubusercontent.com/1562309/77766010-73f0b380-7015-11ea-9187-57cdc2aa8985.png">

**Comment by andreasbergqvist** [comment]
Hmm, this is how my "Open module settings" dialog looks like.
Not sure if we have different versions of Android Studio or if it differs between Windows and Mac.

Did you right click the "app"-folder when selecting "Open module settings"?

![image](https://user-images.githubusercontent.com/1663126/77767006-b976a600-7040-11ea-9af9-51bf44aec699.png)

**Comment by maniart** [comment]
Huh! did you open `./My-RN-Project/android` in Android Studio and then right click on the `android` directory in the file panel on the left side > Open Module Settings?

**Comment by maniart** [comment]
I'm using Android Studio 3.6.1 and when right clicking on the `app` directory, I am not presented with the `Open Module Settings` option at all. Did you have to click on `Make Module` first?

**Comment by maniart** [comment]
Making progress. `File > Sync Project with Gradle files` is now allowing me to find `Open Module Settings` option under `app`

**Comment by peterlofquist** [comment]
Did you get this sorted out?

Gradle & Android Studios is not my realm of expertise. Can you import any `.aar` file to your modules panel?

**Comment by bfarrgaynor** [comment]
Yeah I found these instructions a little unclear too. I was able to add the module, but the part where you go to add the dependency is not accurate and I'm also kind of stuck on it. It seems as though the instructions should be made a little clearer for RN specifically.

**Comment by peterlofquist** [comment]
> Yeah I found these instructions a little unclear too. I was able to add the module, **but the part where you go to add the dependency is not accurate** and I'm also kind of stuck on it. It seems as though the instructions should be made a little clearer for RN specifically.

Could you provide more specific feedback regarding the bold part here?

**Comment by bfarrgaynor** [comment]
There's a gap in the instructions. The first part where you add the module with your 'android' folder focused and the second part where you add the dependency are distinct actions where you open the window, complete the action, then open it again with the 'app' folder focused. Part 1 is android folder, part 2 is 'app' folder. I was able to figure it out, but the instructions make it seem like it's one complete action and I think that's what throws people off. You can't add the dependency in the second part unless you have the app folder focused.
